Add Linux AArch64 wheel build support

odidev opened the pull request:
Related to https://github.com/cgohlke/imagecodecs/issues/24
Add CircleCI support to build Linux AArch64 wheels.

**Note:** aarch64.patch: This can be removed when PR- https://github.com/cgohlke/imagecodecs/pull/25 is merged and released.

odidev commented:
@Czaki, please add this project to CircleCi to trigger CircleCi pipelines.

odidev committed:
Add Linux AArch64 wheel build support

odidev commented:
Fixed few issue. Reopening...

Czaki commented:
I add circleci (with your config, as it is necessary), but after reopen of PR it does not start. I set up a run for forked PR. Maybe You need to add some commit (or rebase)?

odidev commented:
I just un fallowed on my circleci account. reopening to trigger the builds.

odidev committed:
Set CIBW_MANYLINUX_AARCH64_IMAGE to odidev's Image

Will update when official image is released. To make this pipeline pass, just used the image published in odidev's account.

odidev committed:
Merge branch 'master' into aarch64-wheels

odidev commented:
The current pipeline is using [**odidev**/imagecodecs_manylinux2014_aarch64:2021.10.05](https://hub.docker.com/repository/docker/odidev/imagecodecs_manylinux2014_aarch64) image to build the artifacts. I will raise a PR to update the build image when officially published.

odidev commented:
@Czaki, can you please take a look on this PR.

Czaki commented on build_utils/docker_prepare_manylinux2014_aarch64.sh:
```diff
@@ -0,0 +1,18 @@
+yum install -y pcre pcre-devel openssl-devel wget libtool snappy-devel gettext gettext-devel po4a nasm meson cmake3 ninja-build
+
+# On aarch64 machine, yum pull meson==0.47.0 and project requires meson >= 0.49.0 so installing it from source code
+wget https://github.com/mesonbuild/meson/releases/download/0.49.0/meson-0.49.0.tar.gz
+tar -xvzf meson-0.49.0.tar.gz
+cd meson-0.49.0
+export PATH=/opt/python/cp38-cp38/bin/:$PATH
+python setup.py install
+cd ..
+rm -rf meson-0.49.0 meson-0.49.0.tar.gz
```
why this instead of `pip install`? Why this old version instead of the newest one? 
```suggestion
export PATH=/opt/python/cp38-cp38/bin/:$PATH
pip install meson==0.49.0
```

odidev replied:
Installing meson through pip. latest meson is thronging build error. Lets use  meson==0.49.0 for now.

Czaki commented on build_utils/docker_prepare_manylinux2014_aarch64.sh:
```diff
@@ -0,0 +1,18 @@
+cd ..
+rm -rf meson-0.49.0 meson-0.49.0.tar.gz
+
+cp /opt/python/cp38-cp38/bin/meson /usr/bin/meson
+cp /usr/bin/cmake3 /usr/bin/cmake
+cp /usr/bin/ninja-build /usr/bin/ninja
```
```suggestion
ln -s /opt/python/cp38-cp38/bin/meson /usr/bin/meson
ln -s /usr/bin/cmake3 /usr/bin/cmake
ln -s /usr/bin/ninja-build /usr/bin/ninja
```

odidev replied:
Done

Czaki reviewed (changes requested):
Looks good, but I have two questions in `docker_prepare_manylinux2014_aarch64.sh`

Czaki commented:
@odidev ?

odidev commented:
@Czaki, sorry for the delay. I was busy in few other activities. I re-pushed the changes. Can you please review it? Thanks in advance.

Czaki commented:
Thanks

odidev commented:
@Czaki, may I know the plan for releasing AArch64 wheels? Thanks.

Czaki commented:
This is the question to @cgohlke. But as it is only run and upload maybe it will be done fast.  

I think that I should set up building and upload docker images in another repository.

cgohlke commented:
Do you mean the artifacts at <https://app.circleci.com/pipelines/github/Czaki/imagecodecs_build/5/workflows/8beb3653-5363-470c-958e-68355134b881/jobs/5/artifacts>?

Czaki commented:
I see that build is outdated and blosc is not installed in the image. I will try to fix it.
